### PR TITLE
Fix build warning, fix build issue for sparc, and add myself as maint…

### PR DESCRIPTION
**Pull request by katietz**
…ainer

Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ source:
     patches:
       - dont_link_with_libcxxabi.patch
       - 530-ld64-add-conda-specific-env-vars-to-modify-lib-search-paths.patch
+      - fix_sparc_pc.patch
```
Comment (isuruf) [inline recipe/meta.yaml:16]: Can you send this patch to https://github.com/tpoechtrager/cctools-port ?

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml:16]: Sure, I will check, where I can place that patch in that repro. It might be later today

Comment (katietz) [reply recipe/meta.yaml:16]: Done. See https://github.com/tpoechtrager/cctools-port/pull/81

**Comment by isuruf** [comment]
Thanks @katietz. Can you update the version to the latest commit at https://github.com/tpoechtrager/cctools-port/tree/949.0.1-ld64-512.4 ?

**Comment by isuruf** [comment]
Pulled into https://github.com/conda-forge/cctools-and-ld64-feedstock/pull/21. Thanks

**Pull request closed without merging**